.travis.yml

```diff
@@ -30,6 +30,7 @@ script:
   - black --check .
   - flake8 --config=.flake8 .
   - mypy cgp
+  - isort --check-only cgp
```

Comment:
should we check examples and tests as well? why restrict this to the library path?

Reply:
yes, totally, thanks for noticing, I added that.